docs: import external K8s cluster on existing cluster (#11405)
rook/rook: Documentation/CRDs/Cluster/external-cluster.md

  cephVersion:
    image: quay.io/ceph/ceph:v17.2.5 # Should match external cluster version
```

##  Exporting Rook to another cluster

> If you have a multiple K8s cluster running, and want to use `rook-ceph` cluster as the centralize storage place,

Reviewer: How about this paragraph and note?
```suggestion
If you have multiple K8s clusters running, and want to use the local `rook-ceph` cluster as the central storage,
you can export the settings from this cluster with the following steps. 

!!! important
   For other clusters to connect to storage in this cluster, Rook must be configured with a networking configuration that is accessible from other clusters. Most commonly this is done by enabling host networking in the CephCluster CR so the Ceph daemons will be addressable by their host IPs.  
```

For this we need atleast one cluster to running/deploy as `rook-ceph` cluster,
And then you can do the following for other clusters:

1) Copy create-external-cluster-resources.py into the directory `/etc/ceph/` of the toolbox.
   ```console
   toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
   kubectl -n rook-ceph cp deploy/examples/create-external-cluster-resources.py $toolbox:/etc/ceph
   ```

> 2) Exec to the toolbox pod and execute create-external-cluster-resources.py with needed options to create require [users and keys](/Documentation/CRDs/Cluster/external-cluster.md## Supported Features).

Reviewer: Since the link is in the same doc, you just need the place to jump without the doc name.
```suggestion
2) Exec to the toolbox pod and execute create-external-cluster-resources.py with needed options to create required [users and keys](#supported-features).
```

Author: Done thanks

  cephVersion:
    image: quay.io/ceph/ceph:v17.2.5 # Should match external cluster version
```

> ##  Import K8s cluster on existing Rook-Ceph Deployment

Reviewer: How about this subject? And we would need a paragraph describing the scenario. 
```suggestion
##  Exporting Rook to another cluster

<Insert description of the scenario>
```

Author: Added a para

> 1) Copy create-external-cluster-resources.py into /etc/ceph/ directory of toolbox.

Reviewer: Where do they get this script? How about an example command of copying it?
```suggestion
1) Copy create-external-cluster-resources.py into the directory `/etc/ceph/` of the toolbox.
```

Reviewer 2: how about complete adding a complete path of `deploy/examples/create-external-cluster-resources.py`

> 2) SSH into toolbox, execute create-external-cluster-resources.py with needed options to create require users and keys.

Reviewer: ```suggestion
2) Exec to the toolbox pod and execute create-external-cluster-resources.py with needed options to create require users and keys.
```
